packages/markdown/src/lib/deserializer/utils/deserializeMdKeepLineBreak.spec.tsx

```diff
@@ -0,0 +1,160 @@
+/** @jsx jsx */
+
+import { createSlateEditor } from '@udecode/plate-common';
+import { jsx } from '@udecode/plate-test-utils';
+
+import { MarkdownPlugin } from '../../MarkdownPlugin';
+import { deserializeMd } from './deserializeMd';
+
+jsx;
+
+describe('deserializeMd', () => {
+  const editor = createSlateEditor({
+    plugins: [MarkdownPlugin.configure({ options: { splitLineBreaks: true } })],
+  });
+
+  it('should deserialize paragraphs and keep in separate paragraphs with line breaks', () => {
+    const input =
+      'Paragraph 1 line 1\nParagraph 1 line 2\n\nParagraph 2 line 1';
+
+    const output = (
+      <fragment>
+        <hp>Paragraph 1 line 1</hp>
+        <hp>Paragraph 1 line 2</hp>
+        <hp>
+          <htext />
+        </hp>
+        <hp>Paragraph 2 line 1</hp>
+      </fragment>
+    );
+
+    expect(deserializeMd(editor, input)).toEqual(output);
+  });
+
+  it('should deserialize line break tags and keep in separate paragraphs', () => {
+    const input = 'Line 1<br>Line 2';
+    const output = (
+      <fragment>
+        <hp>
+          <htext>Line 1</htext>
```

Comment:
The `<htext>` can be omitted when it's the sole child of a block and has no attributes.